imageio/imageio-psd/src/main/java/com/twelvemonkeys/imageio/plugins/psd/PSDMetadata.java

```diff
@@ -387,6 +387,10 @@ private Node createLayerInfoNode() {
             node.setAttribute("left", String.valueOf(psdLayerInfo.left));
             node.setAttribute("bottom", String.valueOf(psdLayerInfo.bottom));
             node.setAttribute("right", String.valueOf(psdLayerInfo.right));
+            node.setAttribute("layerId", String.valueOf(psdLayerInfo.getLayerId()));
+            node.setAttribute("groupLayerId", String.valueOf(psdLayerInfo.groupLayerId));
+            node.setAttribute("isGroup", String.valueOf(psdLayerInfo.isGroup));
+            node.setAttribute("isSectionDivider", String.valueOf(psdLayerInfo.isSectionDivider));
```

Comment:
Perhaps these attributes could default to the most common value, and be set only if different from the default?